bazel/copts.bzl

```diff
@@ -56,7 +56,28 @@ GRPC_LLVM_WARNING_FLAGS = [
     "-Wno-unused-function",
 ]
 
+# Warning suppression for MSVC on Windows.
+# The list of suppressed warnings should be kept in sync with the cmake build.
+GRPC_MSVC_DEFAULT_COPTS = [
```

Comment:
Could you add comments helpful to see what these warnings are for? something like

```
    "/wd4146" # unary minus operator applied to unsigned type, result still unsigned
    "/wd4200" # nonstandard extension used : zero-sized array in struct/union
```

You can see a list of msvc warnings [here](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warnings-c4000-c5999?view=msvc-170)